src/style/variables.scss

```
@@ -5,6 +5,6 @@ $white: #FFFFFF;
 $blackLight: #252525;
 $darkGrey: #7A7A7A;;
 $lightGrey: rgba(255, 255, 255, 0.125);
-$backgroundColorGlassCard: rgba(17, 25, 40, 0.7); 
+$backgroundColorGlassCard: rgba(18, 26, 41, 0.702); 
```

Wenn es kein Unterschied macht bitte nicht so viele Nachkommastellen
```suggestion
$backgroundColorGlassCard: rgba(18, 26, 41, 0.7); 
```